PowerShellGet/PSModule.psm1

```diff
@@ -13803,6 +13887,61 @@ function Get-PrivateData
     }
 }
 
+function Validate-PreReleaseString 
+{
+    [CmdletBinding(PositionalBinding=$false)]

```

Comment:
Yes, it is meant to be an internal helper function.  For my curiosity, what is CmdletBinding and when should it be used?  I used it here because I was looking at some of the other function definitions as reference.